### PR TITLE
Navigation modal

### PR DESCRIPTION
**Pull request by palash23jain**
Adding modal and navigation bar. Also done some refactoring by adding common main.css and integrating it with all the components

### CODE REVIEW

File: components/modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+                        <pre><code class="language-markup">
+    <div class="navbar-container">
+        <div class="navbar-title">
+            <a href="#">MyWebsite</a>
+        </div>
+        <input type="text" placeholder="search" class="navbar-search-input">
+        <div class="navbar-items">
+            <button class="btn btn-warning">Login</button>
+                <img class="navbar-item" src="heart.svg" alt="">
+                <img class="navbar-item" src="shopping-cart.svg" alt="">
```
Comment (prakash855) [inline components/modal/modal.html]: Add text to alt everywhere

**Review by prakash855 — APPROVED** [review]
Add text to alt in all the image elements, rest is fine!

**Review by SJTGSHIVAM — APPROVED** [review]
I see a nice use of variables! we are good to merge as I don't see any breaking changes and code looks clean.

**Review by devsurajrai — COMMENTED** [review]
Everything looks good.

**Pull request merged**